unity-renderer/Assets/Scripts/Tests/VisualTests/VisualTestsBase.cs

```diff
@@ -8,6 +8,7 @@
 using UnityEngine.Rendering.Universal;
 using Object = UnityEngine.Object;
 
+// test
```

Comment:
```suggestion
```

Reply:
Had to add that line because the CI failed to build with no code changes. Removed it.